src/app/Fake.Core.Tasks/Fake.Core.Tasks.fsproj

```diff
@@ -21,9 +21,7 @@
     <ProjectReference Include="..\Fake.Core.Process\Fake.Core.Process.fsproj" />
     <ProjectReference Include="..\Fake.Core.String\Fake.Core.String.fsproj" />
     <ProjectReference Include="..\Fake.IO.FileSystem\Fake.IO.FileSystem.fsproj" />
-    <ProjectReference Include="..\Fake.Core.Context\Fake.Core.Context.fsproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
+    <ProjectReference Include="..\Fake.Core.Context\Fake.Core.Context.fsproj" FromP2P="true" />
```

Comment:
What does P2P mean here? I hope this has no effect on dependencies.

Reply:
It was already here, i just made the projects nicer.
From what i can tell, it enables the transitive project dependencies to be pulled

Reply:
I actually think all of the references should have it...based on this: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-add-reference

Reply:
P2P = project-to-project